tesseract-clickhouse/src/sql/options.rs

```diff
@@ -76,7 +76,14 @@ pub fn wrap_options(
 
     let mut filters_sql = if !filters.is_empty() {
         let filter_clauses = filters.iter()
-            .map(|f| format!("{} {}", f.by_column, f.constraint.sql_string()));
+            .map(|f| {
+                if let Some(o) = &f.operator {
+                    let c2 = f.constraint2.as_ref().unwrap();
+                    format!("({} {} {} {} {})", f.by_column, f.constraint.sql_string(), o.sql_string(), f.by_column, c2.sql_string())
```

Comment:
can we make the o / f /c2 variables more explicit in terms of what they are meant to represent?